app/articles/models.py

```diff
@@ -0,0 +1,56 @@
+import uuid
+from typing import Any
+
+from cloudinary.models import CloudinaryField
+from django.contrib.auth import get_user_model
+from django.db import models
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+from django.utils.text import slugify
+
+from app.abstracts import TimeStampedModel
+
+User = get_user_model()
+
+
+class Tag(models.Model):
+    name = models.CharField(max_length=50, unique=True)
+
+    def __str__(self) -> str:
+        return self.name
+
+
+class Article(TimeStampedModel):
+    post_id = models.UUIDField(
```

Comment:
should be id only  ideally if you want to override the default one

Reply:
Okay